gst/nnstreamer/tensor_query/tensor_query_serversrc.c

```diff
@@ -308,6 +312,9 @@ gst_tensor_query_serversrc_change_state (GstElement * element,
     case GST_STATE_CHANGE_PAUSED_TO_READY:
       gst_tensor_query_server_remove_data (src->src_id);
       break;
+    case GST_STATE_CHANGE_PAUSED_TO_PLAYING:
+      caps = gst_pad_peer_query_caps (GST_BASE_SRC_PAD (bsrc), NULL);
+      gst_tensor_query_serversrc_set_caps (bsrc, caps);
```

Comment:
need break (may raise svace issue later)

Reply:
Updated!
Thank you :)